examples/lvgldemo/lvgldemo.c

```diff
@@ -153,16 +172,29 @@ int main(int argc, FAR char *argv[])
 
   lv_init();
 
-  /* Display interface initialization */
-
-  fbdev_init();
-
   /* Basic LittlevGL display driver initialization */
```

Comment:
```suggestion
  /* Basic LVGL display driver initialization */
```

Reply:
The projects name is LittlevGL not LVGL, but for consistency I have changed it.

Reply:
I think they recently renamed to LVGL

Reply:
Interesting the front page of the docs still refer to it as LittlevGL anyway I changed it to LVGL everywhere so it is consistent.

Reply:
I think they redefined the acronym to "Light and Versatile Graphics Library"

Reply:
Indeed, they announced the renaming of the project on Release 7.0:
https://blog.lvgl.io/2020-06-01/announcement